client_restaurant/shared/src/commonMain/kotlin/data/gateway/remote/AuthorizationInterceptor.kt

```diff
@@ -10,19 +10,20 @@ import org.koin.core.scope.Scope
 
 fun Scope.intercept(client: HttpClient) {
     client.plugin(HttpSend).intercept { request ->
+
         val localConfigurationGateway = get<LocalConfigurationGateway>()
         val remoteIdentityGateway = get<IdentityRemoteGateway>()
+
         val accessToken = localConfigurationGateway.getAccessToken()
         val refreshToken = localConfigurationGateway.getRefreshToken()
+
         request.headers {
-            append("accessToken", accessToken)
+            append("Authorization", "Bearer $accessToken")
```

Comment:
Is it right way to send token?
@mustafa-sab3alarab

Reply:
yes